test/html-to-react-tests.js

```diff
@@ -101,6 +101,25 @@ describe('Html2React', function() {
 
             assert.equal(reactHtml, htmlExpected);
         });
+
+        it('should parse br elements without warnings', function() {
```

Comment:
I think you should've kept the wording in my PR, e.g. that the test asserts that children aren't created for void elements, as this is more direct.